src/layouts/events-layout.tsx

```diff
@@ -1,48 +1,83 @@
-import { Route, Routes, Outlet } from 'react-router';
 import { useState } from 'react';
-import EventsMain from '../components/events/eventsMain';
+import { Route, Routes, Outlet } from 'react-router';
+import { Alert } from '@mui/material';
+import EventsMain from '../components/events/eventsPage/eventsMain';
 import CreateEvent from '../components/events/createEvent/createEvent';
 import Event from '../components/events/eventFeed/eventFeed';
-import { userRole } from '../types/string-types';
+import { PlatformType, UserRole } from '../types/string-types';
+import { IAlert } from '../types/interfaces';
 
-interface eventProps {
-  userRole: userRole;
+interface Props {
+  userRole: UserRole;
+  platformToShow: PlatformType;
 }
 
-const Events = ({ userRole }: eventProps) => {
-  const [eventToDisplay, setEventToDisplay] = useState<string>();
+const Events = ({ userRole, platformToShow }: Props) => {
+  const [alert, setAlert] = useState<IAlert>({
+    isAlert: false,
+    content: '',
+    severity: 'info',
+  });
+
+  const handleAlert = (
+    isSuccess: boolean,
+    content: string,
+  ) => {
+    setAlert({
+      isAlert: true,
```

Comment:
I think that `true` is not needed when the variable is always truthy

Reply:
It is needed